### ISSUE TITLE
When using a name for a new object which contains a space, the server throws an error

### ISSUE THREAD

**Issue by timcadman** [molgenis/molgenis-service-armadillo#130]
Cause: variable is not trimmed by the server.

ds.make(
  toAssign = "bmi_poc$edu_m+bmi_poc$bmi.730+bmi_poc$sex",
  newobj = "test_ tmp"), 
  datasources = conns
)

**Comment by fdlk** [comment]
We have a regex on the valid symbols, it is in the controller. Is a space allowed in datashield symbol names?

**Comment by timcadman** [comment]
You are allowed to specify variables with a space in I think. Sido/datashield team might be best people to advise on this.

**Comment by mswertz** [comment]
We should check datashield spec.

**Comment by clemens-tolboom** [comment]
Guess this is about

- https://github.com/datashield/DSI/blob/master/R/datashield.assign.R

caused by

- https://github.com/datashield/dsBaseClient using `ds.make`

**Comment by timcadman** [comment]
I agree that you should not be able to specify file names with spaces. However the current error is not very helpful:
`[1] "URL using bad/illegal format or missing URL"`

Could this be improved? Not clear whether error is returned by molgenis-server-armadillo or dsBase.

**Comment by clemens-tolboom** [comment]
Where is the error coming from in the chain

> DataScientist :: DSI :: Armadilli-r :: web :: Armadillo-java :: Profile :: Docker :: rock (session manager) :: rserve (handler)
